biosys/apps/main/api/serializers.py

```diff
@@ -77,7 +77,9 @@ def __call__(self, data):
             validator.schema_error_as_warning = not self.strict
             result = validator.validate(data)
             if result.has_errors:
-                raise ValidationError(list(result.errors.items()))
+                error_messages = ['{col_name}::{message}'.format(col_name=k, message=v) for k, v in
+                                  result.errors.items()]
+                raise ValidationError(error_messages)
```

Comment:
Changed the format of the errors returned by the schema validator. The client should receive a list of string with the format above.